halo2_proofs/examples/cost-model.rs

```diff
@@ -125,7 +125,7 @@ impl FromStr for Lookup {
 
 impl Lookup {
     fn required_degree(&self) -> usize {
-        1 + cmp::max(1, self.input_deg) + cmp::max(1, self.table_deg)
+        2 + cmp::max(1, self.input_deg) + cmp::max(1, self.table_deg)
```

Comment:
I confirmed that the reason for this bug was that this cost model example was implemented before the lookup argument was made perfectly zero-knowledge: https://github.com/zcash/halo2/commit/1f1b705fa662e0ab8cb35e4b5e0fc7fb9cc20282

I also agree that this happens to ensure the lowest returned degree is 4, matching the real code.